src/Particle/ArrayOfStructs.cpp

```diff
@@ -80,7 +80,8 @@ void make_ArrayOfStructs(py::module &m)
 }
 
 void init_ArrayOfStructs(py::module& m) {
-    make_ArrayOfStructs<7,0> (m);
+    make_ArrayOfStructs< 0, 0> (m);
```

Comment:
@RTSandberg this one should cause no problems, does it?
```C++
     make_ArrayOfStructs< 0, 0> (m);
```

Reply:
I can't think of any reason it should cause problems, we even have some tests in place to keep the user from accidentally referencing non-existent real or int data

Reply:
Awesome, thanks! :)